lutris/game.py

```diff
@@ -372,6 +374,9 @@ def do_play(self, prelaunched, error=None):
         if self.runner.system_config.get('disable_compositor'):
             self.desktop_effects(False)
 
+        if self.runner.system_config.get('pre_script'):
+            self.pre_script_thread_start()
```

Comment:
self.pre_script_thread_start() -> self.pre_script_thread.start() ?

Reply:
thats confusing but its actually the name of the fn `pre_script_thread_start`

Reply:
nvm then